ocp_resources/cluster_role_binding.py

```diff
@@ -9,3 +10,27 @@ class ClusterRoleBinding(Resource):
     """
 
     api_group = Resource.ApiGroup.RBAC_AUTHORIZATION_K8S_IO
+
+    def __init__(
+        self,
+        name=None,
+        cluster_role=None,
+        subjects=None,
+    ):
+        super().__init__(name=name)
+        self.cluster_role = cluster_role
+        self.subjects = subjects
+        self.res = None
+
+    def to_dict(self):
+
+        self.res = super().to_dict()
+        self.res.setdefault("roleRef", {})
+        self.res["roleRef"] = {
```

Comment:
cluster_role and subjects can be None, don't you need to check before set them?
Is this valid role without them?

Reply:
cluster_role required, setting to mandatory attribute. 
Subjects aren't required, will add check